cmd/containerboot/settings.go

```diff
@@ -107,7 +107,7 @@ func configFromEnv() (*settings, error) {
 		MetricsEnabled:                        defaultBool("TS_ENABLE_METRICS", false),
 		HealthCheckEnabled:                    defaultBool("TS_ENABLE_HEALTH_CHECK", false),
 		DebugAddrPort:                         defaultEnv("TS_DEBUG_ADDR_PORT", ""),
-		EgressSvcsCfgPath:                     defaultEnv("TS_EGRESS_SERVICES_CONFIG_PATH", ""),
+		EgressProxiesCfgPath:                  defaultEnv("TS_EGRESS_PROXIES_CONFIG_PATH", ""),
```

Comment:
We already have `TS_SERVE_CONFIG`, and it seems possible we will need `TS_INGRESS_PROXY_CONFIG` in the near future, if not others. Is it maybe worth trying to make this new env var a bit more generic so we can reduce some of the options and just have one `TS_PROXY_CONFIG` env var or similar?

Reply:
That env var is used to determine that this proxy is an egress proxy for HA (i.e to determine if we need to run egress-specific logic.), so if we share it, we'd need another mechansim for that and I am not yet sure if we will need to share it

Theoretically we shouldn't need ingress proxy config, since unlike with egress that logic should be covered with the existing serve config and configfile.
If we do need additional config, we'd need to take a look at introducing a 'proxy type' concept, perhaps as a field in the proxy config. But at the moment, I am not sure we will need it.